scripts/pipeline/create_patch_pairaligner_conf.pl

```diff
@@ -204,37 +204,41 @@ =head2 GENERAL CONFIGURATION
 %{$dna_collection->{homo_sapiens_exception}} = ('chunk_size' => 30000000,
 					      'overlap'    => 0,
 					      'include_non_reference' => ($patched_species_is_alignment_reference ? 1 : 0), #include haplotypes
-					      'masking_options' => '"{default_soft_masking => 1}"');
+					      'masking'    => 'soft',
+                                          );
 
 %{$dna_collection->{homo_sapiens_mammal}} = ('chunk_size' => 30000000,
 					     'overlap'    => 0,
 					     'include_non_reference' => ($patched_species_is_alignment_reference ? 1 : 0), #include haplotypes
                                              # if you have a specific selection of repeat elements for the masking
```

Comment:
This comment was related to the `masking_options_file` below - should be removed

Reply:
This view makes my comment above quite unclear - I am referring to line 212/213